Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/dungeon_items_weapons.yml

```
@@ -254,7 +254,8 @@
     offset: 0.0
     rarePrototypes:
     - SpawnDungeonLootGunT4
-    rareChance: 0.15
+    - WeaponRifleMR8 # mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)